Libraries/SDHC/ff13/Source/diskio.h

```diff
@@ -14,9 +14,11 @@ extern "C" {
 #ifdef NATIVE_SDHC
 #include "sdhc_lib.h"
 #endif
-
+#ifdef FLASH
```

Comment:
Arguably we should not modify the `diskio.h` header file at all.  It's provided by FatFS, and it is possible to handle all of the required logic within the implementation file.